.Lib9c.Tests/Action/HackAndSlashTest.cs

```diff
@@ -66,16 +66,17 @@ public HackAndSlashTest()
             }
         }
 
-        [Fact]
-        public void Execute()
+        [Theory]
+        [InlineData(1)]
```

Comment:
I've tried a few things to add that case.

1. Add `[InlineData(GameConfig.RequireClearedStageLevel.ActionsInRankingBoard)]`
2. Execute `HackAndSlass` action from stage 1 to `GameConfig.RequireClearedStageLevel.ActionsInRankingBoard`

In the case of #1, it failed due to the condition of clearing the stages in order.
In the case of #2, I tried to test while waking up from stage 1 to stage `GameConfig.RequireClearedStageLevel.ActionsInRankingBoard` in order, but there was a case of losing the battle in the simulator. 😢

Reply:
https://github.com/planetarium/lib9c/blob/aa7e1e31c797d964a74a0ee3e6264a1c0aebd11f/.Lib9c.Tests/Action/RankingBattleTest.cs#L46-L52

i think it will be helpful for this case. in this case, we just clear stage stageId - 1. and set avatar level enough to clear.

Reply:
Thanks! I'll try.